OpenRA.Mods.Common/Traits/Buildings/Building.cs

```diff
@@ -125,6 +129,9 @@ public IEnumerable<CPos> UnpathableTiles(CPos location)
 
 			foreach (var t in FootprintTiles(location, FootprintCellType.OccupiedUntargetable))
 				yield return t;
+
+			foreach (var t in FootprintTiles(location, FootprintCellType.OccupiedPassableCannotStop))
```

Comment:
Is this correct? If it is, can we please include a comment to explain why we want to be including tiles that claim to be pathable in a method that claims to be returning tiles that aren't pathable?

Reply:
The output of `UnpathableTiles` gives the occupied cells, which needs to include the 'hot' cells in order to be able to have any interaction with locomotors. This method seems to have been named under the assumption that unpathable and occupied are equivalent; this is obviously no longer the case now. It is probably better to just rename the method.

Reply:
Renamed.